code/common/pubsub.q

```diff
@@ -161,27 +161,23 @@ init:{[t]
 
 // Striping data in a TorQ Installation
 // Hash function
-.ds.modmd5:{sum each md5'[string x]}
+.ds.modmd5:{sum each md5'[string x]};
 // use mod to stripe into number of segments
-.ds.map:{[numseg;sym] sym@/:group .ds.modmd5[sym]mod numseg}
+.ds.map:{[numseg;sym] sym!.ds.modmd5[sym]mod numseg};
 
+// Initialise subscription request on startup
+.ds.subreq:()!();
 // Striping function which stores the mappings for any symbols that it has already computed and 
 // for subsequent requests for that symbol, it looks them up
 .ds.stripe:{[input;skey]
     // If no updates, return
     if[0=count sym:distinct input;:()];
```

Comment:
I think you should move the distinct to the next if clause. 
If the if condition here causes the function to return (), then the distinct wasn't required.